src/Umbraco.Examine/ContentIndexPopulator.cs

```diff
@@ -15,7 +16,7 @@ namespace Umbraco.Examine
     /// <summary>
     /// Performs the data lookups required to rebuild a content index
     /// </summary>
-    public class ContentIndexPopulator : IndexPopulator<IUmbracoContentIndex>
+    public class ContentIndexPopulator : IndexPopulator<IUmbracoContentIndex2>
```

Comment:
@nul800sebastiaan that one is breaking change as change behavior of umbraco can we get that PR tagged as breaking change please?